src/renderer/components/PerCurrencySelectAccount/Option.js

```diff
@@ -71,6 +71,11 @@ function AccountRow({
 }) {
   const palette = useTheme("colors.palette");
 
+  const balance =
+    account.type !== "ChildAccount" && account.spendableBalance
+      ? account.spendableBalance
+      : account.balance;
```

Comment:
Can use this https://github.com/LedgerHQ/ledger-live-common/blob/master/src/account/helpers.js#L65

Reply:
But not blocking